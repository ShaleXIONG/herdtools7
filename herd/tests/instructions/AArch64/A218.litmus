AArch64 A218
(* This test should succeed *)
{
 int64_t *p = &x;
 int64_t x = 2;
 0:X0=p;
}
 P0          ;
 LDR X1,[X0] ;
 LDR X2,[X1] ;

locations [0:X1;0:X2;]
forall (true)
