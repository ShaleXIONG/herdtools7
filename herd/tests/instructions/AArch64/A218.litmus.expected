Test A218 Required
States 1
0:X1=x; 0:X2=2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (true)
Observation A218 Always 1 0
Hash=7337f658da576c132ba6a403026111a6

